_posts/en/newsletters/2019-04-02-newsletter.md

```diff
@@ -0,0 +1,269 @@
+
+**Other bech32 sending support updates:** BitGo [announced][bitgo
+segwit] that their API now supports sending to bech32 addresses; see
+their announcement for additional details about bech32 receiving support.
+The Gemini exchange also [apparently][gemini reddit] added bech32
+sending support this week, and users report that Gemini will accept
+deposits to bech32 addresses as well.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], and [Bitcoin Improvement Proposals
+(BIPs)][bips repo].  Note: all merges described for Bitcoin Core are to
+its master development branch; some may also be backported to the
+0.18 branch for the pending 0.18.0 release.*
+
+- [Bitcoin Core #15620][] stops the `maxtxfee` configuration
+  parameter from affecting the `sendrawtransaction` and
+  `testmempoolaccept` RPCs.  Previously those RPCs would default to
+  rejecting a transaction paying a fee higher than the configured max.
+  Now a hardcoded default of 0.1 BTC is used as the acceptable ceiling.
+  The `maxtxfee` configuration parameter is still used by Bitcoin Core's
+  built-in wallet; it has just been separated from node-specific RPCs.
+  This change is part of a general [cleanup of wallet configuration
+  options][Bitcoin Core #13044] as well as part of separating the node
+  and wallet (which both used this setting before this change).
+
+- [Bitcoin Core #15643][] changes the Python script Bitcoin Core
+  developers use to merge commits so that the git merge message includes
+  a list of which developers approved (ACKed) the version of a PR that
+  was merged.  (This internal project change is perhaps not notable by
+  itself, but one of the tool's other features---copying the full PR
+  description into the merge message---makes it much easier for the
+  author of this section to write these merge summaries, so he
+  encourages other Bitcoin projects to investigate the advantages of
+  using this tool for automatically creating better git-based
+  documentation, as well as improving their security and auditability.)
+
+- [Bitcoin Core #15519][] adds a [Poly1305][] implementation to Bitcoin
+  Core but does not use it.  This is expected to be used later for an
+  implementation of [P2P protocol encryption][].
+
+- [Bitcoin Core #15637][] modifies the mempool-related RPCs (such as
+  `getrawmempool`) to rename the `size` field to `vsize`.  The previous
+  value was also the vsize, so the calculation has not changed.  This
+  merged PR simply makes it clear that this is a vsize and not a
+  stripped size.  For backwards compatibility, you can start Bitcoin
+  Core with the `deprecatedrpc=size` parameter to continue using the old
+  field name, although this will be removed in a future release.
+
+- [LND #2759][] lowers the default [CLTV delta][bolt2 delta] from 144
```

Comment:
I think there are some issues with this explanation. I've pushed a commit that I think improves it, but I'm not 100% clear of the exact details of lightning. Can you review and let me know what you think?

Reply:
I think my description was correct.  I'll write an explanation of why, but I suggest that we not publish this unless we get an actual LN expert (such as one of those you @ mentioned) to review whatever version we go with.  (I'll leave it up to you to keep or drop at merge time tomorrow; we can put it in next week's newsletter after we ensure it's accurate.)